website/docs/components/accordion/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Accordion
+description: An accordion allow users to expand or collapse multiple sections of content in a stack.
+caption: Allow users to display multiple sections of content in a stack.
+status: released
+links:
+  figma: {link to the "Accordion" page in the components Figma library}
+  github: {link to the "Accordion" component's folder in the GitHub repo}
+previewImage: assets/illustrations/components/accordion.jpg
+navigation:
+  hidden: false
+  keywords: ['toggle', 'details', 'reveal', 'expand', 'collapse']
```

Comment:
Curious whether it would be useful to include 'list' since many folks conceptualize an accordion item and list item similarly.

Reply:
Good point. Added!